src/validations/rules/poam.sch

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-model schematypens="http://purl.oclc.org/dsdl/schematron" href="../styleguides/sch.sch" phase="basic" title="Schematron Style Guide for FedRAMP Validations" ?>
+<sch:schema
+    queryBinding="xslt2"
+    xmlns:doc="https://fedramp.gov/oscal/fedramp-automation-documentation"
+    xmlns:oscal="http://csrc.nist.gov/ns/oscal/1.0"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron"
```

Comment:
The `xmlns:unit="http://us.gov/testing/unit-testing"` namespace is missing. You recently created it. Why would you not include it?